src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const discount3 = 20;
+  const discount7 = 50;
+  const totalAmount = days * dayCost;
+
+  if (days < 3) {
+    return totalAmount;
+  } else if (days < 7) {
+    return totalAmount - discount3;
+  } else {
+    return totalAmount - discount7;
+  }
 }
```

Comment:
The function name 'calculateRentalCost' is descriptive and clearly indicates its purpose. However, the implementation could be improved by following the checklist.